src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -693,19 +693,10 @@ self =>
     def isLiteralToken(token: Token) = token match {
       case CHARLIT | INTLIT | LONGLIT | FLOATLIT | DOUBLELIT |
            STRINGLIT | INTERPOLATIONID | SYMBOLLIT | TRUE | FALSE | NULL => true
-      case _                                                        => false
+      case _                                                             => false
```

Comment:
The history of this arrow alignment is that it was fixed for interpolation SIP but not interpolation final code. Someone somewhere may have an opinion on formatting and tooling.